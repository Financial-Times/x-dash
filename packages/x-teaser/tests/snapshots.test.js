const { Teaser, presets } = require('../');
const renderer = require('react-test-renderer');
const { data } = require('../stories/article');
const props = { ...data, ...presets.SmallHeavy };

it('renders correctly', () => {
	const tree = renderer.create(Teaser(props)).toJSON();
	expect(tree).toMatchSnapshot();
});
