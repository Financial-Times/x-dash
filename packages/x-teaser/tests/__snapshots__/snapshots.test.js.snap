// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="o-teaser o-teaser--article o-teaser--small o-teaser--has-image o-teaser--highlight js-teaser"
  data-id=""
>
  <div
    className="o-teaser__content"
  >
    <div
      className="o-teaser__meta"
    >
      <div
        className="o-teaser__meta-concept"
      >
        <a
          className="o-teaser__tag"
          data-trackable="teaser-concept"
          href="#"
        >
           Sexual misconduct allegations 
        </a>
      </div>
    </div>
    <div
      className="o-teaser__heading"
    >
      <a
        className="js-teaser-heading-link"
        data-trackable="heading-link"
        href="#"
      >
         Inside charity fundraiser where hostesses are put on show 
      </a>
    </div>
    <p
      className="o-teaser__standfirst"
    >
      FT investigation finds groping and sexual harassment at secretive black-tie dinner
    </p>
    <div
      className="o-teaser__timestamp"
    >
      <time
        className="o-teaser__timestamp-date"
        dateTime="2018-01-23T15:07:00+0000"
      >
        January 23, 2018
      </time>
    </div>
  </div>
  <div
    className="o-teaser__image-container js-teaser-image-container"
  >
    <div
      className="o-teaser__image-placeholder"
      style={
        Object {
          "paddingBottom": "56.2500%",
        }
      }
    >
      <a
        aria-hidden="true"
        data-trackable="image-link"
        href="#"
        tab-index="-1"
        title="Inside charity fundraiser where hostesses are put on show"
      >
        <img
          alt=""
          className="o-teaser__image"
          src="https://www.ft.com/__origami/service/image/v2/images/raw/http%3A%2F%2Fprod-upp-image-read.ft.com%2Fa25832ea-0053-11e8-9650-9c0ad2d7c5b5?source=next&fit=scale-down&compression=best&width=240"
        />
      </a>
    </div>
  </div>
</div>
`;
